aws/ecs_cluster_types.go

```diff
@@ -0,0 +1,35 @@
+package aws
+
+import (
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// ECSClusters - Represents all ECS clusters found in a region
+type ECSClusters struct {
+	ClusterArns []string
+}
+
+// ResourceName - The simple name of the aws resource
+func (clusters ECSClusters) ResourceName() string {
+	return "ecscluster"
+}
+
+// ResourceIdentifiers - the collected ECS clusters
+func (clusters ECSClusters) ResourceIdentifiers() []string {
+	return clusters.ClusterArns
+}
+
+// MaxBatchSize - the maximum number of ECS clusters for a single request
+func (clusters ECSClusters) MaxBatchSize() int {
+	return 200
```

Comment:
Is this a number you found in the API or SDK docs or something?

Reply:
(Might want to clarify in comment)

Reply:
Also would be good to extract magic numbers like this into a const. I.E. at the top of the file, you can have `const MAX_BATCH_SIZE` = 200` (preceded with a comment that explains where that comes from).

Reply:
@brikis98 Very good question - it is something I've been meaning to follow up on. I looked up [here](https://docs.aws.amazon.com/cli/latest/reference/ecs/delete-cluster.html) & [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-quotas.html) & it doesn't seem like there's any mention of the need to rate-limit how many ECS Clusters we delete at one time. 

The rate-limiting here has been added to follow pretty much the same pattern as all other modules in the `aws` package - but I'm more than happy to take guidance on this in terms of reducing or increasing the number. 

@infraredgirl Now that we have a `globals.go` file we could extract this there too! I'd suggest we do that separately, though, as that would mean changing all `*_types.go` files in this PR and in one go, which is probably not ideal. Thoughts?

Reply:
Well, if there's no obvious limit, then I'd prob just take a guess at something reasonable, and add a comment explaining just that. Many AWS APIs are limited to 100 "things," so that might be a good starting point.

Reply:
> @infraredgirl Now that we have a `globals.go` file we could extract this there too! I'd suggest we do that separately, though, as that would mean changing all `*_types.go` files in this PR and in one go, which is probably not ideal. Thoughts?

I don't see a `globals.go` file - did you perhaps forget to `git add` it? At any rate, I think it's totally fine to change it in a follow up PR - just put a `TODO` comment so we don't forget about it.

Reply:
Try pulling the latest - you should see [the file in this commit ](https://github.com/ina-stoyanova/cloud-nuke/blob/1af68c1ef81ebc24def9761405115b2253fadd4a/aws/globals.go#L3)

I will extract this out in the file for now, and will follow up on understanding if the MaxBatchSize for all other AWS modules is arbitrary, and to extract it to also in the `globals.go` file

Reply:
Yep, it's there now! Cool - sounds good. 👍